src/services/completions.ts

```diff
@@ -912,6 +912,7 @@ namespace ts.Completions {
                 completionNodes.push(node);
             },
             body,
+            /*preserveOptional*/ false,

```

Comment:
I think we want this to be true if we are offering a property, and false if we are offering methods. My reasoning is that when we offer a property declaration in completions, we don't also offer an initialization for the property. So all properties added via completions will be uninitialized, and so it makes more sense for them to be optional if the original property was also declared optional. This also minimizes the number of errors that could be introduced by a property declaration.